include/hardware_functions.h

```diff
@@ -19,6 +19,7 @@ void UART_clearScreen(void);
 void UART_Putc(char c);
 void UART_PutStr(char* message);
 void UART_PutInt(uint32_t val);
+uint8_t Get_Strlen(char *string);
```

Comment:
waar gebruik je dit voor

Reply:
Lengte van bericht bepalen zodat ik die mee kan sturen in lora send en zo iets kan versturen via LoRa